manage.py

```diff
@@ -40,10 +43,9 @@ def main():
     run_patches()
 
     from corehq.warnings import configure_warnings
-    configure_warnings(is_testing(sys.argv))
+    configure_warnings()
```

Comment:
Fair question. Should a management command (that is not running tests) fail because of warnings if  `CCHQ_TESTING` is set? For example, if you are running `CCHQ_TESTING=1 ./manage.py migrate` to migrate your test database, should it crash if warnings are emitted? I think that might be more annoying than helpful, but I'm happy to hear arguments in favor.

Reply:
We reckoned that `CCHQ_TESTING=1 ./manage.py dbshell` would useful for inspecting the test database.

Reply:
Are you saying you agree that that command should not crash on a warning? That's my position.